src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,57 @@
+package duke.task;
+
+/**
+ * Class that represents a Task
+ */
+public abstract class Task {
+    /** String containing the description of the Task. */
```

Comment:
Opening /** should be on a separate line. Single line comment should follow something like:
```
/**
 * String containing the description of the Task.
 */
 ```

or can just use // for single line comments